packages/app-bridge/src/react/useAfterInsertion.tsx

```diff
@@ -0,0 +1,39 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { useEffect } from 'react';
+
+import { type AppBridgeBlock } from '../AppBridgeBlock';
+
+/**
+ * Hook to execute a callback after the block has been inserted.
+ * The callback is only executed when the third argument is true (default).
+ *
+ * @example
```

Comment:
Would remove that example here and if you wanna have one add it to the changeset